media/js/firefox/whatsnew/whatsnew-137-video-events.es6.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+function recordInteraction(state) {
+    window.dataLayer.push({
+        event: 'video-interaction',
+        videoTitle: 'wnp-137',
+        interaction: state
+    });
+}
+
+function init() {
+    const video = document.getElementById('wnp-video');
```

Comment:
Do we have existing HTML video event helpers? (the YouTube JS API video clicks are automatically tracked, but HTML video is not) Do we need to check DNT/GCP/cookies here or is that handled by GTM snippet?

This is a stop-gap addition to allow analytics on the upcoming WNP. We could backport this to a more general helper if it's useful.